packages/dom/src/lib/ElementAssertion.ts

```diff
@@ -142,9 +145,45 @@ export class ElementAssertion<T extends Element> extends Assertion<T> {
     );
   }
 
-  private getClassList(): string[] {
-    return this.actual.className.split(/\s+/).filter(Boolean);
-  }
+  /**
+   * Asserts that the element has the specified CSS styles.
+   *
+   * @param expected - The expected CSS styles.
+   * @returns The assertion instance.
+   */
```

Comment:
Can we add the example annotation so users are able to understand how the matcher should be used?
```suggestion
    * @example
   * ```
   * expect(component).toHaveStyle({ color: #ffff });
   * ```
```

Reply:
Sure thing, as I'm seeing others files on the project, we may need to refactor those sections to match the style, such as:

```ts
  /**
   * Description....
   *
   * @example
   * ```
   * expect(foo).toAssert(bar);
   * ```
   *
   * @param foo-params
   * @returns bar-result
   */
```

wdyt